maven/jdk11/Dockerfile.nanoserver

```diff
@@ -1,2 +1,18 @@
-# Use https://github.com/jenkinsci/jnlp-agents as base for now
-FROM jenkins/jnlp-agent-maven:windows-nanoserver-jdk11 AS jnlp
+# escape=`
+FROM mcr.microsoft.com/windows/servercore:1809 as core
+FROM jenkins/inbound-agent:jdk11-nanoserver-1809
+
+# ProgressPreference => Disable Progress bar for faster downloads
+SHELL ["pwsh.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+# https://github.com/StefanScherer/dockerfiles-windows/tree/master/golang-issue-21867
+COPY --from=core C:/windows/system32/netapi32.dll C:/windows/system32/netapi32.dll
+
+ARG MAVEN_VERSION=3.8.6
```

Comment:
Should this be tracked in https://github.com/jenkins-infra/docker-inbound-agents/blob/main/updatecli/updatecli.d/maven.yml too?

Reply:
Yes: will be a subsqeuent PR (with fix on manifests). Let me open an issue to not forget it

Reply:
https://github.com/jenkins-infra/docker-inbound-agents/issues/37